bin/deploy

```diff
@@ -68,7 +68,7 @@ git clean -fxd
 
 # copy documentation
 
-declare -r omnidoc="${base_dir}/target/scala-2.11/omnidoc"
+declare -r omnidoc="${base_dir}/target/scala-2.12/omnidoc"
```

Comment:
Even for the recent release candidates, we are still publishing Scala 2.11 version of scaladocs. This moves it to 2.12, but later we can make this use the default Scala version used by the project (and defined by interplay).